src/components/Selectbox/Selectbox.jsx

```diff
@@ -264,7 +238,10 @@ export default class Selectbox extends React.Component {
 
 	wrapItem(child) {
 		const {theme, selectedItemIconName} = this.props;
-		const isActive = child.props.value && child.props.value === this.state.selectedValue;
+		const {value} = child.props;
+		const {selectedValue} = this.state;
+		const isActive = (typeof value === 'number' || typeof value === 'string') && value === selectedValue;
```

Comment:
Why only numbers and strings here? If you check value existance then it should be done via `typeof value !== 'undefined'`